File: src/main/java/org/taktik/icure/entities/embed/AgreementAppendix.java
package org.taktik.icure.entities.embed;

public class AgreementAppendix {
	private Integer docSeq;
	private Integer verseSeq;
	private String documentId;
	private String path;

	public Integer getDocSeq() {
		return docSeq;
	}

	public void setDocSeq(Integer docSeq) {
		this.docSeq = docSeq;
	}

	public Integer getVerseSeq() {
		return verseSeq;
	}

	public void setVerseSeq(Integer verseSeq) {
		this.verseSeq = verseSeq;
	}

	public String getDocumentId() {
		return documentId;
	}

	public void setDocumentId(String documentId) {
		this.documentId = documentId;
	}

	public String getPath() {
		return path;
	}

	public void setPath(String path) {
		this.path = path;
	}
}

File: src/main/kotlin/org/taktik/icure/entities/embed/AgreementAppendix.kt
package org.taktik.icure.entities.embed

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonInclude
import com.github.pozo.KotlinBuilder
import java.io.Serializable

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonIgnoreProperties(ignoreUnknown = true)
@KotlinBuilder
data class AgreementAppendix(
        val docSeq: Int? = null,
        val verseSeq: Int? = null,
        val documentId: String? = null,
        val path: String? = null
) : Serializable
